src/OpenLoco/Paint/PaintTile.cpp

```diff
@@ -137,9 +141,325 @@ namespace OpenLoco::Paint
         call(0x0048B313, regs);
     }
 
+    struct OffsetAndBBOffset
+    {
+        Map::Pos2 offset;
+        Map::Pos2 boundingOffset;
+    };
+    std::array<OffsetAndBBOffset, 16> _4FE830 = {
+        OffsetAndBBOffset{ { 24, 4 }, { 24, 4 } },
+        OffsetAndBBOffset{ {
```

Comment:
RARRR i spent ages getting this nicely lined up but i must have formatted before commiting!